From 001fa4abe7f4711742f80b07a981a4ccc95b5bdb Mon Sep 17 00:00:00 2001
From: Nate Drude <nate.d@variscite.com>
Date: Thu, 20 Aug 2020 09:03:29 -0500
Subject: [PATCH] imx8mm: Set IMX_BOOT_UART_BASE for console on uart4

i.MX8MM ATF code sets IMX_BOOT_UART_BASE to UART2.
On Variscite i.MX8MM boards, the console is on UART4.
imx-atf will hang on Variscite boards without this change.

Signed-off-by: Nate Drude <nate.d@variscite.com>
---
 plat/imx/imx8m/imx8mm/include/platform_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/imx/imx8m/imx8mm/include/platform_def.h b/plat/imx/imx8m/imx8mm/include/platform_def.h
index 42b03613e..b09b0e6a4 100644
--- a/plat/imx/imx8m/imx8mm/include/platform_def.h
+++ b/plat/imx/imx8m/imx8mm/include/platform_def.h
@@ -51,7 +51,7 @@
 
 #define HAB_RVT_BASE			U(0x00000900) /* HAB_RVT for i.MX8MM */
 
-#define IMX_BOOT_UART_BASE		U(0x30890000)
+#define IMX_BOOT_UART_BASE		U(0x30a60000)
 #define IMX_BOOT_UART_CLK_IN_HZ		24000000 /* Select 24MHz oscillator */
 
 #define PLAT_CRASH_UART_BASE		IMX_BOOT_UART_BASE
